app/services/geolocation/location_lookup_strategy.rb

```diff
@@ -0,0 +1,115 @@
+# frozen_string_literal: true
+
+module Geolocation
```

Comment:
I added comments because this is an interface to be followed. Let me know if you're against and I will remove the commit.